src/pages/Index.vue

```diff
@@ -46,20 +42,45 @@
 
 <script>
 import FormInput from "~/components/FormInput.vue";
-import FormButton from "~/components/Button.vue";
+import FormButton from "~/components/FormButton.vue";
 import BlockLineGroup from "~/components/BlockLineGroup.vue";
 
 export default {
   components: {
     FormInput: FormInput,
-    "v-button": FormButton,
+    FormButton: FormButton,
     BlockLineGroup: BlockLineGroup
   },
-  data: {},
+  data() {
+    return {
```

Comment:
This is pretty good. Probably needs its own section of the return section because you also have to keep track of the block line instances. Maybe this template actually could go into the methods section. If this doesn't make sense you'll see once you start implementing copy to clipboard.

Reply:
@jseanpatel I redesigned.